php/src/Google/Protobuf/Any.php

```diff
@@ -251,7 +251,7 @@ public function setValue($var)
      * targeted message type. If failed, an error will be thrown. Otherwise,
      * the method will create a message of the targeted type and fill it with
      * the decoded value in Any.
-     * @return unpacked message
+     * @return Message unpacked message
```

Comment:
Should this be \Google\Protobuf\Internal\Message

Reply:
The class already has a `use \Google\Protobuf\Internal\Message`, so the namespace is unnecessary.